nncf/common/graph/transformations/commands.py

```diff
@@ -31,17 +34,22 @@ class TargetType(OrderedEnum):
     LAYER = 0
     BEFORE_LAYER = 1
     AFTER_LAYER = 2
-    WEIGHT_OPERATION = 3
+    PRE_LAYER_OPERATION = 3
```

Comment:
Can't see usage of `PRE_LAYER_OPERATION` and `POST_LAYER_OPERATION`. Is it just reserved?
Does it match to PRE_OP and POST_OP in NNCf for Pytorch?

Reply:
Yes, It is reserved for NNCF with Pytorch backend.
NNCF_MODULE_PRE_OP -> PRE_LAYER_OPERATION 
NNCF_MODULE_POST_OP -> POST_LAYER_OPERATION 

The naming is open. Please, provide your variant. Also, I can exclude these types of target points from this PR. @ljaljushkin What do you think?

Reply:
I am ok with naming. I was a bit confused to see duplication of similar entities in NNCF PyTorch, like InsertionType (for TransformationType) and OperationPriority(for TransformationPriority).

It was made intentionally, the next step is to re-implement PyTorch part with new classes, right?

Reply:
Yes, you are right.